Samples/GLFWDemo/assets/RenderStates.json

```diff
@@ -0,0 +1,101 @@
+{
+    "Defaults": {
+        "Shader": {
+            "SourceLanguage": "HLSL",
+            "UseCombinedTextureSamplers": true
+        },
+        "Pipeline": {
+            "PSODesc": {
+                "ResourceLayout": {
+                    "DefaultVariableType": "MUTABLE"
+                }
+            }
+        }
+    },
+    "Pipelines": [
+        {
+            "PSODesc": {
+                "Name": "Generate SDF map PSO",
+                "ResourceLayout": {
+                    "ImmutableSamplers": [
+                        {
+                            "ShaderStages": "COMPUTE",
+                            "SamplerOrTextureName": "g_SrcTex",
+                            "Desc": {
+                                "MinFilter": "LINEAR",
+                                "MagFilter": "LINEAR",
+                                "MipFilter": "LINEAR",
+                                "AddressU": "CLAMP",
+                                "AddressV": "CLAMP",
+                                "AddressW": "CLAMP"
+                            }
+                        }
+                    ]
+                }
+            },
+            "pCS": {
+                "Desc": {
+                    "Name": "Geneate SDF CS"
+                },
+                "FilePath": "GenerateSDF.hlsl",
+                "Macros": [
+                    {
+                        "Name": "RADIUS",
+                        "Definition": "8.0f"
+                    },
+                    {
+                        "Name": "DIST_SCALE",
+                        "Definition": "0.5f"
```

Comment:
You hard-coded these values, but they depend on the constants.
It is better to set them in ModifyCreateInfo